codecov.yml

```
@@ -0,0 +1,5 @@
+# https://docs.codecov.com/docs/ignoring-paths
+ignore:
+  - "source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTests"
+  - "source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.SystemTests"
+  - "source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests"
```

If we go with the ignore paths, let's inlude the tests from `Energinet.Charges.Libaries` as well:

```suggestion
  - "source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests"
  - "source/Energinet.Charges.Libraries/source/Energinet.Charges.Libraries.Clients.Tests"
  - "source/Energinet.Charges.Libraries/source/Energinet.Charges.Libraries.Clients.IntegrationTests"
  - "source/Energinet.Charges.Libraries/source/Energinet.Charges.Libraries.Clients.RegistrationTests"
```